app/components/SegmentedCards.tsx

```diff
@@ -18,8 +18,8 @@ export default function SegmentedCards({
       {children.map((child, index) => (
         <div
           key={index}
-          className={`padding-2 border-base-lighter border-left-2px border-right-2px border-bottom-2px border-solid ${
-            index == 0 ? 'radius-top-md' : ''
+          className={`tablet:height-card padding-2 border-base-lighter border-left-2px border-right-2px border-bottom-2px border-solid full-width-span ${
+            index == 0 ? 'radius-top-md margin-top-1' : ''
```

Comment:
What does this change do?

Reply:
Ah the height is not doing anything anymore, I had originally added it to try to apply a solution for long strings when many notice types were selected, but it is now unnecessary. The second part just gives a little more room between the header row and the content.
This (w/ the margin-top-1) : 
 
![image](https://user-images.githubusercontent.com/11023698/185451381-a522e39a-be3b-4d07-8848-9cb2071e93f6.png)

verses this (w/out):

![image](https://user-images.githubusercontent.com/11023698/185451508-ff501396-f54d-4b1a-ae59-6c830af57775.png)